### DISCUSSION TITLE
Failed to add constraints in MIMIC-IV 1.0

### DISCUSSION THREAD

**Discussion by ZhangAngus** [MIT-LCP/mimic-code#1139] [MIMIC-IV]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Description of the issue, including:
* what you have tried
* references to similar issues
* queries demonstrating your question (if applicable)

### Database
MIMIC-IV, v1.0

When running the `constraint.sql` provided in [https://github.com/MIT-LCP/mimic-code/blob/main/mimic-iv/buildmimic/postgres/constraint.sql](https://github.com/MIT-LCP/mimic-code/blob/main/mimic-iv/buildmimic/postgres/constraint.sql), an error occurred:
```
ERROR:  Insert or update "diagnoses_icd" violates foreign key constraints of "diagnoses_icd_admissions_fk"
Description:  The key value pair (hadm_id)=(20826040) does not appear in the table "admissions".
```
So I checked the `admissions` table. There is no record having hadm_id = 20826040.
And I checked the `admissions.csv.gz` in `MIMIV-IV, v0.4`. There is one record having hadm_id = 20826040.

Please fix the data loss error in new version `admissions` table. Thanks.(●'◡'●)

**Comment by SichengH** [comment]
I would suggest installing MIMIC-IV v1.0 in a new environment, where the previous version v0.4 wouldn't interfere with the installation. Please let me know if that doesn't work. 

Moreover, v0.4 is a testing version and shouldn't be used.

**Comment by epabarker** [comment]
Hi there, I had the same issue with MIMIC-IV v1.0 in a new environment without the previous version. There's also a few more errors similar to the one mentioned above, all related to 20826040. I've ignored these and presumably this is fine seeing as it is one individual? And the fact that constraints aren't necessary due to it being a read only database
